### PR TITLE
Added util.promisify shim dependency for older version of Node

### PR DESCRIPTION
**Pull request by lsparlin**
This is a great tool!  The first thing I did when I found [carbon.now.sh](http://carbon.now.sh) was go see if the was a cli utility for generating them.  When I went to run this it failed on `util.promisify` because it only works on Node 8+.  Mind if I add this shim to make your awesome util work with older Node versions?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const puppeteer = require("puppeteer");
 const qs = require("query-string");
 const globby = require("globby");
 const util = require("util");
+require('util.promisify').shim()
```
Comment (baransu) [inline index.js]: Maybe we can just use polyfill by default
```js
const promisify = require('util.promisify');```

**Review by baransu — COMMENTED** [review]
Whoa! Unexpected PR. I created this project for fun just to play with `pupeeter`. Never expected someone would use it.

Thank you for the PR 😀

**Comment by baransu** [comment]
Also, if it's possible. Please install dependencies using `yarn`. We want consistent `yarn.lock` 😀

**Comment by lsparlin** [comment]
Re: "Never expected someone would use it.": This project was the first result from the google search `"carbon.now generate from cli"` 😃 

I'll make updates to this PR per your suggestions later today

**Review by baransu — APPROVED** [review]

**Comment by baransu** [comment]
LGTM 🎉

**Comment by baransu** [comment]
Will publish as `0.1.3` later today.

**Comment by baransu** [comment]
@lsparlin `0.1.3` is now available on npm: https://www.npmjs.com/package/@baransu/carbon-cli 🎉

**Pull request merged**